watchfiles/run.py

```diff
@@ -250,7 +250,7 @@ def start_process(
             logger.warning('ignoring args and kwargs for "command" target')
 
         assert isinstance(target, str), 'target must be a string to run as a command'
-        popen_args = shlex.split(target)
+        popen_args = shlex.split(target, posix=False)
```

Comment:
```suggestion
        import platform

        posix = platform.uname().system.lower() != 'windows'
        popen_args = shlex.split(target, posix=posix)
```
surely?

Reply:
might be best to move this into a separate function for ease of testing.

Reply:
Made a new function for that...
Could you write test code for the split_cmd function, I don't quite understand the testings

Thanks 😃

Reply:
No, you've created the PR, you should write the tests.

I don't use windows, I don't need this.